src/components/PopUpRevisionContainer/PopUpRevisionContainer.js

```diff
@@ -11,13 +11,14 @@ import useClickOutsideContainer from './../../utils/useClickOutsideContainer'
 import useCloseWithEscapeKey from './../../utils/useCloseWithEscapeKey'
 
 /**
+ * Component that renders the PopUpRevisionContainer component which displays revisions in a timeline form and a overview of revisions.
  *
- * @param {object} props
- * @param {string} titleSingular - Type of object (e.g. Beleidskeuze)
- * @param {object} dataObject - Object we are viewing on the detail page
- * @param {array} revisionObjects - Revisions of the dataObject
- * @param {object} children - Children revision list item component(s)
- * @returns Component to toggle the revision popup and the
+ * @component
+ *
+ * @param {string} titleSingular - Title of the object in a singular form
+ * @param {object} dataObject - Parameter containing the object data.
+ * @param {array} revisionObjects - Parameter containing a list of revisionObjects.
+ * @param {object} children - Parameter containing props that is used within the popup.
```

Comment:
children - can contain child components
See: https://codeburst.io/a-quick-intro-to-reacts-props-children-cb3d2fce4891

Reply:
I've changed the parameter description.

Reply:
See [this comment](https://github.com/Provincie-Zuid-Holland/Omgevingsbeleid-Frontend/pull/556/files/80ec8f36399d84cca8d3eb96529a4c7a134a2be2#r677272472)